schema/schema.sql

```diff
@@ -0,0 +1,63 @@
+-- Create Item class table
+CREATE TABLE item (
+  id int PRIMARY KEY GENERATED ALWAYS AS IDENTITY,
+  genre text,
+  author text,
+  label text,
+  publish_date date,
+  archived boolean,
+);
+
+-- Create Music Album class table
+CREATE TABLE music_album (
+  id SERIAL PRIMARY KEY,
+  item_id int FOREIGN KEY REFERENCES item(id),
+  on_spotify boolean,
+  publish_date date,
+  archived boolean,
+);
+
+-- Create a Genre class table
+CREATE TABLE genre (
+  id SERIAL PRIMARY KEY,
+  item_id int FOREIGN KEY REFERENCES item(id),
+  name varchar
+);
+
+
+-- Game class table
+CREATE TABLE game (
+  id SERIAL PRIMARY KEY ,
+  item_id INT FOREIGN KEY REFERENCES item(id),
+  multiplayer VARCHAR,
+  last_played_at DATE,
+  publish_date DATE,
+);
+
+-- Author class table
+CREATE TABLE author (
+  id SERIAL PRIMARY KEY,
+  item_id INT FOREIGN KEY REFERENCES item(id),
+  first_name VARCHAR,
+  last_name VARCHAR
+);
+
+
+-- Books
+CREATE TABLE books (
+  id SERIAL PRIMARY KEY,
+  publisher VARCHAR,
+  cover_state VARCHAR,
+  genre_id INT NULL REFERENCES genres(id),
+  author_id INT NULL REFERENCES authors(id),
+  label_id INT NULL REFERENCES labels(id),
+  publish_date DATE NOT NULL,
+  archived BOOLEAN NOT NULL
+);
+
+-- Labels
+CREATE TABLE labels (
+  id SERIAL PRIMARY KEY,
+  title VARCHAR(150) NOT NULL,
+  color VARCHAR(150) NOT NULL
+);
```

Comment:
Kindly have a reference to the author, genre,  source and label tabels from books, music and games table. I have recorded [this video](https://www.loom.com/share/3bb2fbd21a324177b1ba95212895e1bc) where I explained in detail what you need to fix.